pages/Category/components/Products/index.jsx

```diff
@@ -32,114 +29,16 @@ class Products extends Component {
   };
 
   /**
-   * Initializes the Products component.
-   * @param {Object} props The components props.
-   */
-  constructor(props) {
-    super(props);
-
-    this.gridContainer = null;
-    this.listContainer = null;
-
-    /**
-     * Variable to make sure that only one of the two ProductXxx Components
-     * is rendered in the first place.
-     */
-    this.initialRender = true;
-  }
-
-  /**
-   * Once the component did mount and it renders for the first time
-   * we have to set initialRender to false. So that the second ProductXxx components gets
-   * rendered the next time.
-   */
-  componentDidMount() {
-    this.initialRender = false;
-  }
-
-  /**
-   * When the view mode changes apply the updated inline styles directly.
-   * @param {Object} nextProps The components new props.
-   */
-  componentWillReceiveProps(nextProps) {
-    if (this.props.viewMode !== nextProps.viewMode) {
-      this.applyInlineStyle(nextProps);
-    }
-  }
-
-  /**
-   * The component should not update when the viewmode is changed.
-   * A different view mode will only cause a different inline style.
-   * It is not necessary to re-render the whole tree again.
+   *
    * @param {Object} nextProps The components new props.
    * @returns {boolean}
    */
   shouldComponentUpdate(nextProps) {
```

Comment:
Please remove the next props parameter, since it's not used within the function. This is treated as an error.

Reply:
You didn't read my comment :-( Since the function always returns `true`, i think it makes sense to simply remove it.